pkg/cmd/onepasswordtemplatefuncs.go

```diff
@@ -248,12 +248,14 @@ func (c *Config) onepasswordGetOrRefreshSessionToken(args *onepasswordArgs) (str
 		sessionToken = os.Getenv("OP_SESSION_" + args.account)
 
 		switch {
-		case c.Onepassword.version.Major < 2:
+		case c.Onepassword.version.Major == 1:
 			commandArgs = []string{"signin", args.account, "--raw"}
-		case c.Onepassword.version.Major < 3:
+		case c.Onepassword.version.Major >= 2:
```

Comment:
I had intentionally chosen _not_ to do `>= 2` because it is possible that Agilebits will change the protocol again for v3. We’re several years _away_ from it since v2 was just released, but it felt safer to put upper boundaries on compatibility.

In fact, it _might_ be worth doing the `panic(&unsupportedVersionError{…})` during version detection so that it errors out earlier.